pkgs/development/python-modules/scipy/default.nix

```diff
@@ -1,34 +1,41 @@
 {
   lib,
-  stdenv,
-  fetchFromGitHub,
-  fetchpatch,
   fetchurl,
   writeText,
-  xcbuild,
   python,
   buildPythonPackage,
+  fetchFromGitHub,
+  fetchpatch,
+  stdenv,
```

Comment:
I also don't see much value in moving these - putting `stdenv` 2nd makes sense to me.

Reply:
This I was not sure about. I would be fine with having it a the top of you prefer.
I don't know if there is any guideline for this specific item.